pkg/virt-handler/device-manager/device_plugin_base.go

```diff
@@ -26,23 +26,25 @@ import (
 	"net"
 	"os"
 	"path"
-	"path/filepath"
 	"sync"
 	"time"
 
-	"github.com/fsnotify/fsnotify"
 	"google.golang.org/grpc"
 	"kubevirt.io/client-go/log"
 
 	pluginapi "kubevirt.io/kubevirt/pkg/virt-handler/device-manager/deviceplugin/v1beta1"
 )
 
+type healthFunc func() error
+
+type registerFunc func()
```

Comment:
nit: it isn't necessary to declare a separate type. The function signature is simple enough that you can directly declare it in the struct. E.g.
```golang 
registerFunc func()
healthcheck func()
```
It makes the declaration more compact and save a couple of lines

Reply:
done